### ISSUE TITLE
RangeError: Array size is not a small enough positive integer.

### ISSUE THREAD

**Issue by Electroid** [oven-sh/bun#3567]
When running the postinstall script for `puppeteer`, there appears to be an error.

```
$ cd node_modules/puppeteer && bun install.js
```

```js
155 | 
156 |   var width = Math.min(this.width, availableSpace);
157 | 
158 |   /* TODO: the following assumes the user has one ':bar' token */
159 |   completeLength = Math.round(width * ratio);
160 |   complete = Array(Math.max(0, completeLength + 1)).join(this.chars.complete);
                 ^
RangeError: Array size is not a small enough positive integer.
      at /node_modules/progress/lib/node-progress.js:160:13
      at /node_modules/progress/lib/node-progress.js:97:2
      at /node_modules/@puppeteer/browsers/lib/cjs/CLI.js:230:8
      at onData (/node_modules/@puppeteer/browsers/lib/cjs/httpUtil.js:96:12)
      at r (node:stream:2:67910)
      at a (node:stream:2:66676)
      at node:http:447:8
```

**Comment by redjonzaci** [comment]
Can I assist with this?

**Comment by Electroid** [comment]
Looks like this is because of #1774

**Comment by Electroid** [comment]
Implemented in 728c8fdcdb4e9276a7bd0721bad3b5d0cbed67fa and will be available in the next release, v0.7, or now if you run: `bun upgrade --canary`
